pyproject.toml

```diff
@@ -1,3 +1,155 @@
+[build-system]
+requires = ["setuptools >= 61.1.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
```

Comment:
[nit] Do you think we should also set `license` attribute in this directive?

```
license = {text = "MIT License"}
```

Reply:
Ah, it makes sense, thank you. Added in 46b248c (using `file` property though).
https://peps.python.org/pep-0621/#license